src/lib/PaymentElement.svelte

```diff
@@ -15,6 +15,7 @@
   const dispatch = createEventDispatcher()
 
   onMount(() => {
+    register(stripe)
```

Comment:
Also should this file also you the `getContext('stripe')` you employ in the other components? 🙂

Reply:
Good question. My reasoning for `<PaymentElement/>` not needing a `<Container/>` was that when folks use `<PaymentElement/>` it will be the only component they use, so they can pass `stripe` as a prop directly to `<PaymentElement/>`.
Let me know if that feels confusing. Maybe it's better to keep things the same across the board even if it results in more code? I could go either way :slightly_smiling_face: